src/os/bluestore/BlueStore.cc

```diff
@@ -9118,10 +9118,10 @@ void BlueStore::_kv_sync_thread()
       }
 
       {
-	utime_t finish = ceph_clock_now();
-	utime_t dur_flush = after_flush - start;
-	utime_t dur_kv = finish - after_flush;
-	utime_t dur = finish - start;
+        coarse_mono_time finish = coarse_mono_clock::now();
```

Comment:
i'd suggest avoid s/tab/<spaces>/ if possible, it just distracts the reviewer.

Reply:
I completely agree with that. But since most of Ceph code uses spaces, I figured I'd switch to spaces here too since I'm touching these lines. I'll revert this.